### PR TITLE
cba/prepare 2.1.9

### PR DESCRIPTION
**Pull request by christianbader**

### CODE REVIEW

File: blackbox/docs/release_notes/2.1.9.txt [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+.. _back up your data: https://crate.io/a/backing-up-and-restoring-crate/
+
+Changelog
+=========
+
+Fixes
+-----
+
+- Admin UI improvements.
```
Comment (joemoe) [inline blackbox/docs/release_notes/2.1.9.txt]: Didn't we want to make them explicit in this list? Copy them over from the Admin UI release?

File: blackbox/docs/release_notes/2.1.9.txt [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  results.
+
+- Fixed the problem that ``stats.breaker.*`` settings were not applied to the
+  correct circuit breaker.
+
+- Fixed an issue that caused an error ``Primary key value must not be NULL``
+  to be thrown when trying to insert rows in a table that has a generated
+  column which is used both in ``PARTITION_BY`` and ``PRIMARY KEY``.
+
+- The postgres wire protocol service can now be bound to IPv6 addresses as
```
Comment (joemoe) [inline blackbox/docs/release_notes/2.1.9.txt]: is this really a fix?

Comment (mfussenegger) [reply blackbox/docs/release_notes/2.1.9.txt]: Yes because it was documented that it is possible but it didn't work

Comment (joemoe) [reply blackbox/docs/release_notes/2.1.9.txt]: ok, understand

**Pull request merged**